sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -161,9 +162,15 @@ func App() *cli.App {
 					Destination: &useCachedImage,
 					Value:       false,
 				},
+				&cli.StringFlag{
```

Comment:
A [StringSliceFlag](https://cli.urfave.org/v2/examples/flags/#multiple-values-per-single-flag) that lets you pass one label per instance of `--label` might be better here.